fix from upstream cvs

----------------------------
revision 1.245
date: 2010-08-13 22:50:14 -0400;  author: psmith;  state: Exp;  lines: +1 -1;  commitid: 4UaslPqQHZTs5wKu;
- Add oneshell to $(.FEATURES) (forgot that!)

Index: main.c
===================================================================
RCS file: /sources/make/make/main.c,v
retrieving revision 1.244
retrieving revision 1.245
diff -u -p -r1.244 -r1.245
--- main.c	10 Aug 2010 07:35:34 -0000	1.244
+++ main.c	14 Aug 2010 02:50:14 -0000	1.245
@@ -1138,7 +1138,7 @@ main (int argc, char **argv, char **envp
      a macro and some compilers (MSVC) don't like conditionals in macros.  */
   {
     const char *features = "target-specific order-only second-expansion"
-                           " else-if shortest-stem undefine"
+                           " else-if shortest-stem undefine oneshell"
 #ifndef NO_ARCHIVES
                            " archives"
 #endif
